common/static/common/js/discussion/utils.js

```diff
@@ -386,8 +386,7 @@
                         /*
                          bug fix, ordering is off
```

Comment:
As I get it, it seems like someone intentionally turned off ordering of mathjax but we can not find the trace in git blame.  [TNL-5451](https://openedx.atlassian.net/browse/TNL-5451) was created as it was expecting the correct ordering. 

So, It's the matter of a question that whether or not we need ordering in Mathjax. If we do this https://github.com/edx/edx-platform/pull/13551 is going to fix it. If not we can simply close the ticket. 

@adampalay We need to communicate with someone in TNL to get the answer.